apps/frontend/src/lib/chip/leaderboard.ts

```diff
@@ -42,6 +42,59 @@ export async function updateLeaderboardEntry(
   }
 }
 
+export async function updateWorkoutLeaderboardEntry(): Promise<void> {
```

Comment:
Could naming of this feature be changed to "WorkoutCheckIns"? I may also have a workout feature of some sort from more Strava data so it'd be nice to have more specific / differentiated naming.

Reply:
this is under /chip, so would only refer to chip based leaderboard updates